src/components/gltf-model.js

```diff
@@ -0,0 +1,61 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var GLTFLoader = THREE.glTFLoader; // To appease the linter
+var loader = new GLTFLoader();
+var camera = null;
+var gltf = null;
+
+module.exports.Component = registerComponent('gltf-model', {
+  schema: {
+    type: 'src'
+  },
+
+  init: function () {
+    this.model = null;
+  },
+
+  update: function () {
+    var self = this;
+    var el = this.el;
+    var src = this.data;
+
+    camera = this.el.sceneEl.camera;
+    if (!src) { return; }
+
+    this.remove();
+
+    loader.load(src, function (gltfModel) {
+      var i = null;
+      var len = null;
+      var animation = null;
+
+      gltf = gltfModel;
+
+      self.model = gltfModel.scene;
+      el.setObject3D('mesh', self.model);
+      el.emit('model-loaded', {format: 'gltf', model: self.model});
+
+      if (gltf.animations && gltf.animations.length) {
+        len = gltf.animations.length;
+        for (i = 0; i < len; i++) {
+          animation = gltf.animations[i];
+          animation.loop = true;
+          animation.play();
+        }
+      }
+    });
+  },
+
+  remove: function () {
+    if (!this.model) { return; }
+    this.el.removeObject3D('mesh');
+  },
+
+  tick: function () {
+    if (gltf && camera) {
+      THREE.glTFAnimator.update();
```

Comment:
I tried commenting it out, and the animation doesn't play.  If the THREE.AnimationMixer is being called elsewhere (I assume it is), then that would mean this is also required.

Moving it out of the component into the main event loop does make sense, but would make more work later to decouple the glTF functionality in a mix-and-match component system.